assets/src/edit-story/components/richText/provider.js

```diff
@@ -80,9 +81,20 @@ function RichTextProvider({ children }) {
   // This filters out illegal content (see `getFilteredState`)
   // on paste and updates state accordingly.
   // Furthermore it also sets initial selection if relevant.
+  const lastKnownStyle = useRef(null);
```

Comment:
Could this be moved up before `setStateFromContent` for readability? Since it's used in there but defined only here.